crawl4ai/async_configs.py

```
@@ -447,13 +467,22 @@ def __init__(
         if self.browser_type in ["firefox", "webkit"]:
             self.channel = ""
             self.chrome_channel = ""
+        if proxy:
+            warnings.warn("The 'proxy' parameter is deprecated and will be removed in a future release. Use 'proxy_config' instead.", UserWarning)
```

Ruff (0.13.1)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)